packages/warehouses/src/warehouseClients/WarehouseBaseSqlBuilder.ts

```diff
@@ -10,8 +10,6 @@ import { getDefaultMetricSql } from '../utils/sql';
 export default abstract class WarehouseBaseSqlBuilder
     implements WarehouseSqlBuilder
 {
-    abstract readonly type: WarehouseTypes;
```

Comment:
This was causing trouble because it doesn't exist on `WarehouseClient` plus it's not needed!